include/circt/Dialect/FIRRTL/Transforms/Passes.td

```diff
@@ -4,17 +4,17 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef CIRCT_DIALECT_FIRRTL_PASSES_TD
-#define CIRCT_DIALECT_FIRRTL_PASSES_TD
+#ifndef CIRCT_DIALECT_FIRRTL_TRANSFORMS_PASSES_TD
+#define CIRCT_DIALECT_FIRRTL_TRANSFORMS_PASSES_TD
 
 include "mlir/Pass/PassBase.td"
 
-def LowerFIRRTLTypes : Pass<"lower-firrtl-types", "firrtl::FModuleOp"> {
+def LowerFIRRTLTypes : Pass<"firrtl-lower-types", "firrtl::FModuleOp"> {
```

Comment:
The naming change sounds good to me. The previous name was in line with the old `lower-to-rtl` pass, but the new name seems better in the long run..